metals/src/main/scala/scala/meta/internal/tvp/MetalsTreeViewProvider.scala

```diff
@@ -15,69 +15,20 @@ import scala.meta.internal.mtags.GlobalSymbolIndex
 import scala.meta.internal.mtags.Mtags
 import scala.meta.internal.mtags.Symbol
 import scala.meta.internal.semanticdb.Scala._
+import scala.meta.internal.semanticdb.SymbolOccurrence
 import scala.meta.io.AbsolutePath
 
 import ch.epfl.scala.bsp4j.BuildTarget
 import ch.epfl.scala.bsp4j.BuildTargetIdentifier
 import org.eclipse.{lsp4j => l}
 
 class MetalsTreeViewProvider(
-    workspace: () => AbsolutePath,
+    getTrees: () => List[MetalsTreeFolderViewProvider],
```

Comment:
```suggestion
    getFolderTreeViewProviders: () => List[MetalsTreeFolderViewProvider],
```
bit long, but otherwise it might be confused with scalameta trees or compiler ones.